forecast-hub-report.Rmd

```diff
@@ -25,15 +35,12 @@ source("load-data.R")
 
 # evaluation options
 models <- "all" # models to evaluate
-root_dir <- "../data-processed" # root directory for the submission files
+ # root directory for the submission files
+root_dir <- "../covid19-forecast-hub-de/ data-processed"
```

Comment:
white space before 'data-processed'

Reply:
mate I don't think you can talk to me about white space.... 

I would suggest merge and code clean up or not merge and pull out these changes in a tidy way.